src/families/tron/AccountBalanceSummaryFooter.js

```diff
@@ -150,4 +162,41 @@ const styles = StyleSheet.create({
   },
 });
 
-export default AccountBalanceSummaryFooter;
+function getInfoCandidates(t: TFunction): { [key: InfoName]: ModalInfo[] } {
+  const currency: CryptoCurrency = (findCryptoCurrencyById("tron"): any);
```

Comment:
all of this could be put inside const variables outside of a function or component scope since its called in the component without a memo it will be reprocessed i guess :man_shrugging:

Reply:
oh yea, I was using memo before but I changed this func to custom hook but useTranslation doesn't exported so I put it back and forgot to useMemo with this func.

Reply:
please use `getCryptoCurrencyById` which is the "less safe" equivalent but that would prevent you to cast as `any` and ignore the induced flow error. agreed with the outside scope.

Reply:
Cool! i didn't know about this function (getCryptoCurrencyById). I'll replace. I casted as any with the same reason I described here (https://github.com/LedgerHQ/ledger-live-mobile/pull/1181#discussion_r396332325).

Reply:
fixed: https://github.com/LedgerHQ/ledger-live-mobile/pull/1181